tests/unit/core/components/TestAMInstallButton.js

```diff
@@ -393,19 +393,52 @@ describe(__filename, () => {
     const enable = sinon.spy();
     const install = sinon.spy();
 
-    const root = render({ addon, enable, install });
+    const root = render({
```

Comment:
We should probably update the previous test case too (line 380 of that file)

Reply:
yep i missed that one, thanks!